specs/UI_import_gluster_cluster_workflow.adoc

```diff
@@ -0,0 +1,196 @@
+= Import cluster workflow
+
+The intent of import cluster workflow is to provide a mechanism to add a gluster storage cluster into tendrl for management.
+
+
+== Problem description
+
+This specification describes import cluster workflow which is a mechanism to add an gluster storage cluster into tendrl for management.
+
+== Use Cases
+
+This addresses the use case of importing existing gluster clusters into tendrl for management.
+
+The workflow is as follows:
+
+* API will provide the list of nodes and the clusters available for import to the Tendrl.
+
+* User will click on the import button for the cluster, he wants to import.
+
+* A popup will be opened for it.
+
+* UI will filter the available nodes for importing the cluster.
+
+* The pop will list down the available nodes and a checkbox for enabling/disabling the cluster wide volume profiling.
+
+* Each node's name and IP address will be shown. 
+
+* User can checked/unchecked if he wants to enable/disable the volume profiling.
+
+* All the available nodes will participate in the import cluster workflow.
+
+* User clicks the import button form the popup to begin the import operation.
+
+* Success message will be shown on the popup once import operation successfully started.
+
+* Clicking on cancel button will take the user to cluster list page.
```

Comment:
Are we planning to show the task progress page? If the task is successfully submitted.